src/Compilers/CSharp/Portable/Symbols/Tuples/TupleTypeSymbol.cs

```diff
@@ -493,17 +493,39 @@ internal static string TupleMemberName(int position)
             return "Item" + position;
         }
 
+        private static bool IsElementNameForbidden(string name)
+        {
+            switch (name)
+            {
+                case "CombineHashCodes":
```

Comment:
I think members that are internal to the VT implementation and are not meant for public consumption should be removed from the list.